geospace-http/src/main/resources/application.conf

```diff
@@ -37,6 +37,15 @@ com.socrata {
       domains = ["regions.test.socrata.com"]
       bounding-shape-precision = 0.99
     }
+
+    partitioning {
+      # Controls the size of a single partition of region shapes.  Shapes are loaded one partition at a time,
+      # as many point datasets only use a subset of large national shapes like zip codes.
+      # These are in WGS84 coords, so sizeX corresponds to longitude and sizeY corresponds to latitude
+      # At the 45th parallel, 5 deg longitude = 78km and 5 deg latitude = 111 km.
+      sizeX = 5.0
```

Comment:
Hmm... I think I accidentally took the calculation for ONE degree of lat/lon.  I think that means we should probably make each partition smaller, maybe 3 deg by two deg, which roughly translates into a 220km (wide) x 220km (tall) patch.